### DISCUSSION TITLE
README 변경 건

### DISCUSSION THREAD

**Discussion by yoonucho** [OttSideProject/OnDe-FE#6] [Ideas]
안녕하세요!
README 파일이 기본 상태로 되어 있는 점이 계속 마음에 걸려서요. 
우선 제가 저희 프로젝트에 맞게 대략적으로 수정해보려고 하는데, 괜찮을까요? 
여러분의 의견이 궁금합니다. 의견 주시면 감사하겠습니다!

**Comment by yewon830** [comment]
아 넵! 좋습니다!

**Comment by keemewoon** [comment]
넵 저도 좋습니다. 감사합니다 (_ _)

**Comment by yoonucho** [comment]
제가 feature/update-readme 브랜치를 생성하여 작업 후 PR 드릴게요! 우선 제가 작업하던 영역이 있어서, 그 부분을 어느 정도 마무리한 뒤 진행하도록 하겠습니다! 😸
